lib/diagnostic.js

```diff
@@ -29,29 +39,46 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
-
+const any = function (workingArray, callback) {
+  let index = workingArray.findIndex(callback);
+  if (index < 0) {
+    return false;
+  }
+  return true;
 };
 
+
 // Question 4
 // Write a function that returns a new array
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.filter`
-const select = function (/* array, callback */) {
-
+const select = function (workingArray, callback) {
+  const newArry = new Array(workingArray.length);
+  workingArray.forEach((element, index, array)=> {
+    newArray[index] = callback(element, index, array);
```

Comment:
We only want the elements where invoking the callback function on it returns a truthy value.